### DISCUSSION TITLE
App requests to add and upload a specified photo!

### DISCUSSION THREAD

**Discussion by yu0o** [immich-app/immich#13524] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A lot of people have a habit of uploading a given photo to a server. But Immich doesn't have this feature. It only automatically backs up a certain photo album, which causes me to upload it on the web. This is very cumbersome. A lot of folks don't like switching between apps and web pages. I would like to add this feature. This is important to me

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The app does support uploading individual files. The timeline shows all the files from the albums selected in the backup settings, and you have the option to manually trigger an upload of those by selecting them if they haven't been uploaded by the automatic process already.

**Reply by yu0o** [reply]
Thank you for your answer.Is there any hope of adding the ability to upload individual files in the future?My friends and I don't want to use this feature on web pages.

**Reply by yu0o** [reply]
可以添加 自定义或者短链接分享吗 比如 域名/XXXX的
